PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1600,8 +1603,9 @@ bool Notepad_plus::findInFiles()
 	_findReplaceDlg.putFindResult(nbTotal);
 
 	FindHistory & findHistory = (NppParameters::getInstance())->getFindHistory();
-	if (nbTotal && !findHistory._isDlgAlwaysVisible)
+	if ((nbTotal > 0) && !findHistory._isDlgAlwaysVisible)

```

Comment:
I assume you're trying to catch the (assumed) failure condition where `nbTotal` is `-1`?

`Nice catch!`